build.sbt

```diff
@@ -6,7 +6,7 @@ def previousVersion = "0.7.0"
 def scala213 = "2.13.6"
 def scala212 = "2.12.15"
 def scala211 = "2.11.12"
-def scala3 = "3.0.1"
+def scala3 = "3.1.0"
```

Comment:
This will force munit users to upgrade their Scala 3 version as well :/ not sure there's a better way though

Reply:
I set 3.1.0 because Scala Native 0.4.3-RC1 was released only for this version. However, `3.1.1` is still needed for macro mixing and 3.1.1 should have `scala-target` (I think that's how they called it) compiler flag which will allow targeting `3.0.2`.

Reply:
Aha, got it. Maybe we can set 3.1.1 for the native build only and build with 3.0 for the other targets? Otherwise it's a breaking change for jvm and js users

Reply:
If I understand correctly there shouldn't be any breaking changes if we'll set `-scala-release 3.0` (that's how it'll be called)

Reply:
Is that available already? I thought it was still in the works

Reply:
Sorry, you mentioned it in the first comment and I totally missed it 😫 it's late for me I suppose

Reply:
`-scala-release` hasn't been release in `3.1.1` and I decided to use your advice, I set `3.1.1` only for native build.